source/cloud/azure/azure-ml-multi-node.md

```diff
@@ -0,0 +1,72 @@
+# Azure ML Compute cluster
+
+Launch Azure's [ML Compute cluster](https://learn.microsoft.com/en-us/azure/machine-learning/how-to-create-attach-compute-cluster?tabs=python) to distribute your RAPIDS training jobs across single or multi-GPU compute nodes. The Compute cluster scales up automatically when a job is submitted, and executes in a containerized environment, packaging your model dependencies in a Docker container.
+
+## Pre-requisites
+
+All you need to get started is an Azure machine learning workspace and development environment must be installed with [Python SDK v2](https://learn.microsoft.com/en-us/python/api/overview/azure/ai-ml-readme?view=azure-python)or [Azure CLI v2](https://learn.microsoft.com/en-us/azure/machine-learning/how-to-configure-cli?tabs=public). You could install Azure Machine Learning on your local computer but is recommended to use Azure's compute instance that is fully configured with all necessary packages and tools.
+
+### Instantiate workspace
+
+If using the Python SDK, connect to your workspace either by explicitly providing the workspace details or load from `config.json` file.
+Make sure to replace `subscription_id`,`resource_group`, and `workspace_name` with your own.
+
+```console
+# Get a handle to the workspace
+ml_client = MLClient(
+    credential=DefaultAzureCredential(),
+    subscription_id="<SUBSCRIPTION_ID>",
+    resource_group_name="<RESOURCE_GROUP>",
+    workspace_name="<AML_WORKSPACE_NAME>",
+)
+
+# or load details from config file
+ml_client = MLClient.from_config(credential=DefaultAzureCredential(),
+                                path="path_to_config_file",
+                                )
+```
+
+### Create AMLCompute
+
+You will need to create a compute target using Azure ML managed compute (AmlCompute) for remote training. Note: Be sure to check limits within your available region. This [article](link) includes details on the default limits and how to request more quota.
+
+`size`: The VM family of the nodes, specify compute targets from one of `NC_v2`, `NC_v3`, `ND` or `ND_v2` GPU virtual machines in Azure (e.g `Standard_NC12s_v3`)
+
+`max_instances`: The max number of nodes to autoscale up to when you run a job
+
+NOTE:
+You may choose to use low-priority VMs to run your workloads. These VMs don't have guaranteed availability but allow you to take advantage of Azure's unused capacity at a significant cost savings. The amount of available capacity can vary based on size, region, time of day, and more.
+
+```console
+gpu_compute = AmlCompute(
+    name="gpu-cluster",
+    type="amlcompute",
+    size="STANDARD_NC12S_V3",
+    max_instances=2,
+    idle_time_before_scale_down=300,
+    tier="low_priority", # optional
+)
+ml_client.begin_create_or_update(gpu_compute).result()
```

Comment:
Once I run this it's unclear what I need to do. How do I use the cluster? How do I delete it again?

I see the following output in the notebook.

```
AmlCompute({'type': 'amlcompute', 'created_on': None, 'provisioning_state': 'Succeeded', 'provisioning_errors': None, 'name': 'jtomlinson-gpu-cluster', 'description': None, 'tags': None, 'properties': {}, 'print_as_yaml': True, 'id': '/subscriptions/fc4f4a6b-4041-4b1c-8249-854d68edcf62/resourceGroups/rapids-deployment-doc/providers/Microsoft.MachineLearningServices/workspaces/skirui-azureml-rapids/computes/jtomlinson-gpu-cluster', 'Resource__source_path': None, 'base_path': '/mnt/batch/tasks/shared/LS_root/mounts/clusters/jtomlinson1/code', 'creation_context': None, 'serialize': <msrest.serialization.Serializer object at 0x7f37edd3dd90>, 'resource_id': None, 'location': 'westus2', 'size': 'STANDARD_NC12S_V3', 'min_instances': 0, 'max_instances': 2, 'idle_time_before_scale_down': 300.0, 'identity': None, 'ssh_public_access_enabled': True, 'ssh_settings': None, 'network_settings': <azure.ai.ml.entities._compute.compute.NetworkSettings object at 0x7f37edd3d850>, 'tier': 'low_priority', 'enable_node_public_ip': True, 'subnet': None})
```

Reply:
I also don't see RAPIDS mentioned anywhere in this setup. Will this compute cluster have Dask, cudf, etc?

Reply:
The RAPIDS libraries are installed via the docker container that is used as the environment  to run copies of the training script across the cluster nodes. 

I'm working on adding more details, thanks!